DataFormats/Common/interface/ELseverityLevel.h

```diff
@@ -0,0 +1,135 @@
+#ifndef DataFormats_Common_ELseverityLevel_h
+#define DataFormats_Common_ELseverityLevel_h
+
+// ----------------------------------------------------------------------
+//
+// ELseverityLevel.h - declare objects that encode a message's urgency
+//
+//	Both frameworker and user will often pass one of the
+//	instantiated severity levels to logger methods.
+//
+//	The only other methods of ELseverityLevel a frameworker
+//	might use is to check the relative level of two severities
+//	using operator< or the like.
+//
+// ----------------------------------------------------------------------
+
+#include <string_view>
+#include <cassert>
+
+namespace edm {
+
+  // ----------------------------------------------------------------------
+  // ELseverityLevel:
+  // ----------------------------------------------------------------------
+
+  class ELseverityLevel {
+  public:
+    // ---  One ELseverityLevel is globally instantiated (see below)
+    // ---  for each of the following levels:
+    //
+    enum ELsev_ {
+      ELsev_noValueAssigned = 0  // default returned by map when not found
+      ,
```

Comment:
I understand this is a verbatim copy, but would it be worth to reformat
```suggestion
      ELsev_noValueAssigned = 0,  // default returned by map when not found
```
?

Reply:
done